.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  root: true,
+  // This tells ESLint to load the config from the package `eslint-config-custom`
```

Comment:
Wouldn't it be better to also define global linting rules to enforce more uniform WCs and auxiliary code?

Reply:
Yes! Recommend method is to have the separate package for the same! But I didn't want to make this pr more complex! Hence it will be the separate issue. Similarly for the Typescript transpiler we would have separate package.